build.gradle

```diff
@@ -11,42 +11,55 @@ buildscript {
     dependencies {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${spring_boot_version}"
         classpath "io.spring.gradle:propdeps-plugin:0.0.10.RELEASE"
-        classpath "org.openapitools:openapi-generator-gradle-plugin:3.3.0"
-        classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.0.0"
-        classpath "com.github.ben-manes:gradle-versions-plugin:0.21.0"
         // this allows us to find outdated dependencies via './gradlew dependencyUpdates'
+        classpath "com.github.ben-manes:gradle-versions-plugin:0.21.0"
         //jhipster-needle-gradle-buildscript-dependency - JHipster will add additional gradle build script plugins here
     }
 }
 
 plugins {
-    id "org.sonarqube" version "2.7"
+    id "java"
+    id "maven"
+    id "war"
+    id "idea"
+    id "jacoco"
+    id "com.google.cloud.tools.jib" version "0.9.11"
+    id "com.gorylenko.gradle-git-properties" version "2.0.0"
+    id "com.github.node-gradle.node" version "1.3.0"
     id "net.ltgt.apt-eclipse" version "0.21"
     id "net.ltgt.apt-idea" version "0.21"
     id "net.ltgt.apt" version "0.21"
-    id "io.spring.dependency-management" version "1.0.7.RELEASE"
     id "com.moowork.node" version "1.3.1"
 //    id 'org.liquibase.gradle' version "2.0.1"
     id "com.github.ben-manes.versions" version "0.21.0"
     //jhipster-needle-gradle-plugins - JHipster will add additional gradle plugins here
     id "com.diffplug.gradle.spotless" version "3.21.1"
 }
 
-apply plugin: 'java'
 sourceCompatibility=1.8
 targetCompatibility=1.8
-// Until JHipster supports JDK 9
-assert System.properties['java.specification.version'] == '1.8'
+assert System.properties["java.specification.version"] == "1.8" || "11" || "12"
```

Comment:
Sweet! Java 8 has changed its license, and newer version of Ubuntu do not have OpenJDK 8 anymore, not having it as requirement anymore is a great step ahead!